apps/tup-ui/src/pages/Projects/Projects.tsx

```diff
@@ -22,9 +22,16 @@ const Layout: React.FC = () => {
       <section className={styles['project-section']}>
         <PageLayout
           top={
+          <>
             <SectionHeader actions={<NewProject />} isNestedHeader>
               Projects & Allocations
             </SectionHeader>
+            {import.meta.env.VITE_FEATURE_PROJECTS_ENABLED &&
+              <SectionMessage type="error">
+                Maintenance, <time>November 4, 9AM - 1PM Central</time>: Projects & Allocations are not available.
```

Comment:
Project Mainteance is Tues Nov 4, 9AM - 1PM CT. You will be unable to manage your project users during this time. See https://tacc.utexas.edu/news/user-updates/107601/

Reply:
Thanks. Message changed in b4ab0ba.